ports/afx/portfile.cmake

```
@@ -0,0 +1,29 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO EnzoMassyle/AudioFX
+    REF ${VERSION}
+    SHA512 f056d4821e4ad185b80e6926610a28a0a0499c9817b8d639bcdefb19f02e6e865e70364a182952de8ab9b23cc0515c2c3ff5925ebe2fd25c4b0079b4157a9251
+    HEAD_REF main
+)
+
+if(VCPKG_TARGET_IS_ANDROID)
+    set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE "$ENV{ANDROID_NDK_HOME}/build/cmake/android.toolchain.cmake")
+endif()
+
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+)
+
+vcpkg_cmake_install()
+
+vcpkg_cmake_config_fixup(PACKAGE_NAME "AFX" CONFIG_PATH "share/cmake/AFX")
```

```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH "share/cmake/AFX")
```
Let it use the default PACKAGE_NAME, the port name `afx`. It only controls the target directory name, and lower case is fine for cmake.